internal/services/costmanagement/export_resource_base.go

```diff
@@ -104,6 +111,12 @@ func (br costManagementExportBaseResource) arguments(fields map[string]*pluginsd
 							"TheLast7Days",
 						}, false),
 					},
+
+					"data_version": {
+						Type:         pluginsdk.TypeString,
+						Optional:     true,
```

Comment:
I don't think this should be optional. I'm pretty sure it's required unless the API is defaulting.